pkg/deploy/strategy/support/lifecycle.go

```diff
@@ -375,6 +375,11 @@ func makeHookPod(hook *deployapi.LifecycleHook, rc *kapi.ReplicationController,
 
 	gracePeriod := int64(10)
 
+	podSecurityContext, err := kapi.Scheme.DeepCopy(rc.Spec.Template.Spec.SecurityContext)
```

Comment:
@mfojtik @kargakis so the pod SC and the container SC can differ.  The lifecycle hook targets a container to get the image.  Do you think it is reasonable to use the container SC as well?  Ie. set both the pod SC and container SC or just the pod SC.